torch/csrc/inductor/aot_inductor_interface.h

```diff
@@ -1,8 +1,50 @@
-#include "aot_inductor_interface.h"
```

Comment:
This probably wouldn't work for the internal uses, where the definitions of the interfaces are included in two places:

* the generated .so (via ```#include <aot_inductor_interface.h>```; and
* the runtime that includes <aot_inductor_interface.h>

I am thinking perhaps we could rename aot_inductor_interface.h to aot_inductor_interface_impl.h while keeping the original aot_inductor_interface.h, if we don't want to have any wrapper code with a .cpp suffix.

Reply:
Good catch. I think in the generated .so, we only needs to include aot_inductor_model.h because that's that part we will generate code as its definition. aot_inductor_interface.h|cpp should be generated as a part of libtorch, and we will also need take care of the compatibility issues for them using the same mechanism we have discussed.

Reply:
I probably should commit from fbcode to make sure things work there.

Reply:
> Good catch. I think in the generated .so, we only needs to include aot_inductor_model.h because that's that part we will generate code as its definition. aot_inductor_interface.h|cpp should be generated as a part of libtorch, and we will also need take care of the compatibility issues for them using the same mechanism we have discussed.

Hmm, if the generated .so doesn't contain interface.h/cpp, how would the runtime dlopen this .so and call the C APIs?

Reply:
> I probably should commit from fbcode to make sure things work there.

Yes, it's probably a good idea :)